CHANGELOG.unreleased.md

```diff
@@ -12,6 +12,7 @@
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
 - [Import/VM] Fix `UUID_INVALID(VM, OpaqueRef:...)` error when importing from URL
+- [Github] Fix `error message` field not being correctly formatted when creating new issue with bug report form template (PR [7357](https://github.com/vatesfr/xen-orchestra/pull/7357))
```

Comment:
It's not a bad idea but we usually only add XO user changes to the CHANGELOG, not stuff related to devs or GitHub repo, you can remove :slightly_smiling_face: